public/docs/_examples/hierarchical-dependency-injection/dart/lib/hero_card_component.dart

```diff
@@ -0,0 +1,16 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'hero.dart';
```

Comment:
@kwalrath why do you think relative imports are bad? I haven't seen any comment that they should be avoided, also not when library statements are omitted.
Would't that make Dart look more cumbersome?